src/udev/udev-event.c

```diff
@@ -720,10 +720,12 @@ int udev_build_argv(struct udev *udev, char *cmd, int *argc, char *argv[]) {
 
         pos = cmd;
         while (pos != NULL && pos[0] != '\0') {
-                if (pos[0] == '\'') {
-                        /* do not separate quotes */
+                if (pos[0] == '\'' || pos[0] == '"') {
```

Comment:
why not :)

Reply:
Regarding your failing test case: good catch ! actually in the test cases "%s" needs to be escaped as well otherwise udev will do its string substitutions.

I'll fix that.

Thanks !